source/extensions/filters/http/cdn_loop/BUILD

```diff
@@ -24,3 +39,20 @@ envoy_cc_library(
         "//source/common/common:statusor_lib",
     ],
 )
+
+envoy_cc_extension(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    security_posture = "unknown",
```

Comment:
I've got the fuzzer PR ready for review in PR #13183.  I've been running it on my workstation since Friday and the fuzzer hasn't reported anything yet!

What's the best way to do perf testing?